nixos/modules/services/blockchain/ethereum/lighthouse.nix

```diff
@@ -47,6 +47,12 @@ in {
               '';
             };
 
+            disableUpnp = mkOption {
+              type = types.bool;
+              default = true;
+              description = lib.mdDoc "Enables UPNP.";
```

Comment:
If the option is `disableUpnp`, then it would make more sense to me for the description to be "Disables UPNP," and maybe expand the acronym?